### DISCUSSION TITLE
How do I apply styling to certain tiles?

### DISCUSSION THREAD

**Discussion by AbbosRakhmonov** [wojtekmaj/react-calendar#582] [Q&A] [answered]
I need some help I have to list important events that should take place on special days at the bottom of the calendar, according to the information in the backend. So how can I change the color of special days and when I click on that day some information should appear below.
All this is done using react. Thanks in advance for the reply.

**Comment by wojtekmaj** [comment, marked as answer]
Use `tileClassName` for applying class names to certain tiles, and use this classname to change the color and/or apply other styles to said tiles.

Example: https://codesandbox.io/s/react-calendar-tileclassname-0nlmh

Similarly, you can use `tileContent` to add extra content to any tile.

Example: https://codesandbox.io/s/react-calendar-tilecontent-b5bfg

To trigger an action when you click on a specific day, you should use `onDayClick`.

Example: https://codesandbox.io/s/react-calendar-ondayclick-h74t3

**Reply by AbbosRakhmonov** [reply]
Thank you so much :-)
